roles/pulp/defaults/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+pulp_image: pulp/pulp:stable
+pulp_ports:
+  - "8080:80"
+pulp_volumes:
```

Comment:
nothing creates those paths in `/var/lib/pulp`, so the deployment fails to mount them

Reply:
Thanks for addressing this by adding a task.